packages/openid4vc/src/shared/models/index.ts

```diff
@@ -32,7 +32,7 @@ export type OpenId4VciIssuerMetadata = OpenId4VciIssuerMetadataV1Draft11 | OpenI
 export type OpenId4VciIssuerMetadataDisplay = MetadataDisplay
 
 export type OpenId4VciCredentialRequest =
-  | CredentialRequestJwtVcJson
+  | (CredentialRequestJwtVcJson & { credential_definition: { type: string[] } })
```

Comment:
I think we forgot to update the credential request type when adding support for V13. 

I think we can do: 
```
export type OpenId4VciCredentialRequest = UniformCredentialRequest
```

UniformCredentialRequest contains both draft 11 and draft 13 types. And for the others we can then do:

```
export type OpenId4VciCredentialRequestJwtVcJson = CredentialRequestJwtVcJson | CredentialRequestJwtVcJsonV1_0_13
```

and also for all the other format specific ones